src/pages/index.js

```diff
@@ -16,42 +18,82 @@ import {
 const editFormElement = document.querySelector("#profile-edit-modal");
 const addFormElement = document.querySelector("#add-card-modal");
 const profileEditBtn = document.querySelector("#profile-edit-button");
+const avatarEditBtn = document.querySelector("#profile-image-button");
+const AvatarEditForm = document.forms["edit-avatar-form"];
```

Comment:
`AvatarEditForm`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)